Sming/Components/Network/src/Network/Http/Resource/HttpEventedResource.cpp

```diff
@@ -0,0 +1,126 @@
+#include "HttpEventedResource.h"
+#include "../HttpServerConnection.h"
+
+static constexpr char SKIP_HEADER[]{2, 1, 0};
+
+HttpEventedResource::HttpEventedResource(HttpResource* resource)
+{
+	originalResource.reset(resource);
+
+	// [ Register the main events with priority 0 ]
+	addEvent(
+		EventType::EVENT_URL, [this](HttpServerConnection& connection, char** at = nullptr, int* length = nullptr) {
```

Comment:
optional param values not necessary here (`= nullptr`)

Reply:
@mikee47 Thank you Mikee!